lib/histogram.js

```diff
@@ -64,6 +65,7 @@ class Histogram {
 
 		this.name = config.name;
 		this.help = config.help;
+		this.aggregate = config.aggregator || AGGREGATORS.SUM;
```

Comment:
Originally I had it there, but then it has to be repeated where the 2nd constructor form (formal arguments) is handled:

```js
if (isObject(name)) {
  // once here
} else {
  config = {
    // again here
  }
}
```